frontend/src/scenes/data-warehouse/editor/outputPaneLogic.ts

```diff
@@ -22,4 +23,11 @@ export const outputPaneLogic = kea<outputPaneLogicType>([
             },
         ],
     }),
+    urlToAction(({ actions }) => ({
+        '*': (_, __, hashParams) => {
```

Comment:
You'll want to scope this down to just the SQL editor path - otherwise it can cause weirdness on other scenes